RecoLocalFastTime/FTLCommonAlgos/interface/MTDTimeCalib.h

```diff
@@ -0,0 +1,32 @@
+#ifndef RecoLocalFastTime_FTLCommonAlgos_MTDTimeCalib_H
+#define RecoLocalFastTime_FTLCommonAlgos_MTDTimeCalib_H 1
+
+#include "DataFormats/ForwardDetId/interface/MTDDetId.h"
+
+#include "Geometry/MTDGeometryBuilder/interface/MTDGeometry.h"
+#include "Geometry/MTDNumberingBuilder/interface/MTDTopology.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+class MTDTimeCalib
+{
+ public:
+  //constructor & destructor
+  MTDTimeCalib(edm::ParameterSet const& conf, const MTDGeometry* geom, const MTDTopology* topo);  
+  virtual ~MTDTimeCalib() {};
```

Comment:
there are no derived classes inheriting from `MTDTimeCalib`, so the destructor does not need to be virtual